packages/terra-form-validation/src/terra-dev-site/doc/form-validation/ValidationFunctions.4.doc.mdx

```diff
@@ -1,109 +1,26 @@
-
-**Example validation functions**
-
-```js
-const required = value => (value ? undefined : 'Required')
-
-const minLength = value => {
-  const MIN_LENGTH = 3;
+import { Badge } from 'terra-form-validation/package.json?dev-site-package';
 
-  if (value === undefined || value === null) {
-    return 'Empty Strings are not allowed'
-  }
+import ValidationAsynchronous from '../example/ValidationAsynchronous?dev-site-example';
+import ValidationParsing from '../example/ValidationParsing?dev-site-example';
 
-  if (value.length < MIN_LENGTH) {
-    return `String must be at least ${MIN_LENGTH} characters`;
-  }
+<Badge />
 
-  return '';
-}
-```
+# Validation Functions
 
-In order to use them, you just need to provide the validation prop on the react-final-form Field component, and populate the desired input component with the onChange and value provided through the Field validation library, like in the example usage below:
-
-```jsx
-import { Field } from 'react-final-form';
-import InputField from 'terra-form-input/lib/InputField';
-
-<Field
-  type="text"
-  name="user_name"
-  validate={required}
->
-  {({ input, meta, placeholder, ...rest }) => (
-    <InputField
-      {...rest}
-      inputId="description"
-      label="Description"
-      error={meta.error}
-      isInvalid={meta.submitFailed}
-      inputAttrs={{
-        placeholder: 'Description',
-        ...input,
-      }}
-      required
-
-      onChange={(e) => { input.onChange(e.target.value); }}
-      value={input.value}
-    />
-  )}
-</Field>
-```
+*react-final-form* gives you the ability to create your own validation functions. For custom validations, the idea is that a message will be returned if the function finds an error with the form input value, or returns undefined if no errors are detected for the given input value.
 
 ## Asynchronous Validations
 
 There maybe times you need to do remote validations such as Unique Username checks. To best handle this scenario with wide browser support, an OnBlurValidation can be added to a component, where the function takes in two parameters: the value of the input, and a callback setError that sets the error status of the input.
 
 **Note**: If you need to run this in IE11, the [transform-async-to-generator](https://www.npmjs.com/package/babel-plugin-transform-async-to-generator) plugin needs to be added to your .babelrc file.
 
-**Example Asynchronous Validation**
-
-```jsx
-const validateUniqueUser = async (name) => {
-  let response = new Promise((resolve, reject) => {
-    if (name !== 'Unique User') {
-      return resolve('');
-    } else {
-      return resolve('Name is Unavailable');
-    }
-  });
-
-  await response;
-  return response;
-}
-
-```
-
-*Note:* The promise can be any kind of promise, including calls with axios and other libraries.
+**Note**: The promise can be any kind of promise, including calls with axios and other libraries.
```

Comment:
IMO, We could have a single "Note" with bullet points rather than having 2 separate notes

Reply:
Changed e52f755